src/modules/nft/data_provider/history.rs

```diff
@@ -75,13 +76,8 @@ mod tests {
         let block = get_block();
         let contract = near_primitives::types::AccountId::from_str("x.paras.near").unwrap();
         let token = "293708:1";
-        let pagination = types::query_params::HistoryPagination {
-            block_height: block.height,
-            block_timestamp: block.timestamp,
-            limit: 10,
-        };
 
-        let history = get_nft_history(&pool, &contract, token, &pagination).await;
+        let history = get_nft_history(&pool, &contract, token, &block, 10).await;
```

Comment:
I would discourage the usage of "magic numbers" (`10`)

Reply:
and other places below

Reply:
I think it's OK to use constants in tests

Reply:
I am sorry, I've missed those are tests, reading the code was so exciting I've missed the plot twist :)